src/polymer-language-server.ts

```diff
@@ -284,5 +294,18 @@ handleErrors<Result, Fallback>(
   }
 }
 
+function log(val: any) {
+  if (!connection) {
+    return;
+  }
+  if (typeof val !== 'string') {
+    val = util.inspect(val);
+  }
+  connection.console.log(val);
+}
+if (Math.random() > 10000) {
+  log('To ensure that log does not warn about being unused.');

```

Comment:
It's not referenced because I deleted the references, but it's super useful when debugging problems, so I figured why not leave it here for the next time someone's debugging an issue.